sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -10918,7 +10961,135 @@
           },
           "properties": {},
           "lockKey": true,
-          "customConditional": "show = data.dependantStatus === \"independant\";"
+          "customConditional": "show data.dependantstatus === \"independant\";"
```

Comment:
The condition is broken.  But the condition must be updated to `show = !!data.dependantstatus;` to show when `show = !!data.dependantstatus` has any output value.
![image](https://github.com/user-attachments/assets/05e98869-597c-425b-ae19-0d8e27cf247b)